joint_trajectory_controller/test/test_trajectory_controller_utils.hpp

```diff
@@ -0,0 +1,313 @@
+// Copyright 2020 Open Source Robotics Foundation, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef TEST_TRAJECTORY_CONTROLLER_UTILS_HPP_
```

Comment:
:+1: for splitting those files.

Although, this complicates the review...

Reply:
Yeah I'm sorry, I should've probably separated, commited and then started the refactoring there.